set.mm

```diff
@@ -704635,6 +704646,264 @@ exists a proof for (a and b).  (Contributed by Jarvin Udandy,
 $)
 $( Skip $[ set-main.mm $] $)
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  General auxiliary theorems (1)
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  The empty set - extension
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    $d X y $.
+    $( The empty set is the one and only set which is a subset of every set.
+       (Contributed by AV, 24-Aug-2022.) $)
+    al0ssb $p |- ( A. y X C_ y <-> X = (/) ) $=
+      ( cv wss wal c0 wceq 0ex sseq2 ss0b syl6bb 0ss ax-gen sseq1 albidv mpbiri
+      spcv impbii ) BACZDZAEZBFGZTUBAFHSFGTBFDUBSFBIBJKQUBUAFSDZAEUCASLMUBTUCAB
+      FSNOPR $.
+  $}
+
+  ${
+    $d x y A $.  $d x y B $.
+    raaan2.1 $e |- F/ y ph $.
+    raaan2.2 $e |- F/ x ps $.
+    $( Rearrange restricted quantifiers with two different restricting classes,
+       analogous to ~ raaan .  It is necessary that either both restricting
+       classes are empty or both are not empty.  (Contributed by Alexander van
+       der Vekens, 29-Jun-2017.) $)
+    raaan2 $p |- ( ( A = (/) <-> B = (/) )
+                   -> ( A. x e. A A. y e. B ( ph /\ ps ) <->
+                        ( A. x e. A ph /\ A. y e. B ps ) ) ) $=
+      ( c0 wceq wb wa wn wo wral rzal adantr wne df-ne sylbir dfbi3 adantl nfcv
+      pm5.1 syl12anc r19.28z ralbidv nfral r19.27z sylan9bbr jaoi sylbi ) EIJZF
+      IJZKUMUNLZUMMZUNMZLZNABLDFOZCEOZACEOZBDFOZLZKZUMUNUAUOVDURUOUTVAVBVDUMUTU
+      NUSCEPQUMVAUNACEPQUNVBUMBDFPUBUTVCUDUEUQUTAVBLZCEOZUPVCUQFIRZUTVFKFISVGUS
+      VECEABDFGUFUGTUPEIRVFVCKEISAVBCEBCDFCFUCHUHUITUJUKUL $.
+  $}
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+    Unordered and ordered pairs - extension for singletons
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    $d x y z $.
+    $( There is a unique element of a singleton which is equal to another
+       singleton.  (Contributed by AV, 24-Aug-2022.) $)
+    eusnsn $p |- E! x { x } = { y } $=
+      ( vz csn wceq weu weq wal wex equequ2 bibi2d albidv cvv sneqbg elv ax-gen
+      cv wb spei df-eu mpbir ) AQZDBQZDEZAFUDACGZRZAHZCIUGUDABGZRZAHCBCBGZUFUIA
+      UJUEUHUDCBAJKLUIAUIAUBUCMNOPSUDACTUA $.
+  $}
+
+  ${
+    $d x y $.
+    $( If the class abstraction ` { x | ph } = { y } ` of a wwf ` ph ` is a
+       singleton, the wwf is true for the singleton element ` y ` .
```

Comment:
wwf --> wff (twice).  And I would write: either
> the class abstraction { x | ph } associated with the wff ph

or
> the extension { x | ph } of the wff ph

Reply:
OK

Reply:
Done